components/ai_chat/resources/page/components/conversation_list/index.tsx

```diff
@@ -124,6 +125,11 @@ function ConversationList(props: ConversationListProps) {
           const isAIAssistant = turn.characterType === mojom.CharacterType.ASSISTANT
           const showSiteTitle = id === 0 && isHuman && shouldSendPageContents
 
+          // It's assumed that previousTurn represents the user's last input,
+          // and an actionType that should summarize the page.
+          const previousTurn = conversationHistory[id-1]
+          const showLongPageContentInfo = previousTurn?.actionType === mojom.ActionType.SUMMARIZE_PAGE && isAIAssistant && context.shouldShowLongPageWarning
```

Comment:
Why would we only show the long page content warning when the action is for summarize? That seems to be a departure from what we currently do.